src/MovieDetails.js

```diff
@@ -15,7 +13,7 @@ const details = props.details;
           <p> {details.tagline} </p>
           <p> {details.overview} </p>
         </div>
-        <img className="details-poster" src={details.poster_path} />
+        <img className="details-poster" src={details.poster_path} alt={details.title}/>
```

Comment:
Thank you for adding alt text here! I feel foolish for forgetting it in the first draft.